osu.Game/Rulesets/Mods/Mod.cs

```diff
@@ -91,10 +91,13 @@ public virtual string SettingDescription
         [JsonIgnore]
         public virtual bool HasImplementation => this is IApplicableMod;
 
+        public virtual bool IsPlayable(ModUsage usage) => true;
+
         [JsonIgnore]
+        [Obsolete("Override IsPlayable instead.")] // Can be removed 20220918
```

Comment:
This obsoletion path is really weird because if a consumer has set `UserPlayable => false` then that spec will now be essentially ignored. Usually the new method would delegate to the old in the interim (with a warning suppression) until the consumers have migrated to the new one and it is removed.

Reply:
Right, missed to realize that.